internal/cli/usercmd/app.go

```diff
@@ -618,9 +618,17 @@ func (c *EpinioClient) AppRestage(appName string) error {
 	}
 
 	log.V(3).Info("stage response", "response", stageResponse)
-
 	stageID := stageResponse.Stage.ID
+
 	log.V(1).Info("start tailing logs", "StageID", stageID)
+	c.stageLogs(app.Meta, stageID)
+
+	log.V(1).Info("wait for job", "StageID", stageID)
+	// blocking function that wait until the staging is done
+	_, err = c.API.StagingComplete(app.Meta.Namespace, stageID)
+	if err != nil {
+		return errors.Wrap(err, "waiting for staging failed")
+	}
```

Comment:
You can just return the error with Wrap (if `nil` it will return `nil`).

Reply:
done